### ISSUE TITLE
...more of a suggestion/request

### ISSUE THREAD

**Issue by Jagohu** [ThomDietrich/miflora-mqtt-daemon#116]
First of all a big thanks for your work!
If it's not much hassle, it'd be nice to be able to address any of the sensors listed in the config.ini for individual checks too. For example you don't need everything to be checked that often, only one.
$1 could give the name to be checked.
Thank you for your consideration!

**Comment by ThomDietrich** [comment]
Hey there! I also realized that the cyclic check as it is implemented now is not very good. The project started as a small script and grew out of proportion over time.

Your request is already covered in #111 in points 2.4/3.4

In other words: Yes this is planned for but will only be available with the new code base. This change wouldn't be easy to add now. Workaround: If your aim is to query different sensors in different intervals you could run two instances of the daemon in parallel (two separate folders, two separate configs, two separate service files).
